src/screens/AddSiteFormScreen.js

```diff
@@ -274,7 +269,7 @@ class AddSiteFormScreen extends Component {
                     <Text h2
                           style={headerTitle}
                     >
-                        {'Site Image'}
+                        {site_image}
```

Comment:
Going to have to keep cleaning up stuff like this on consecutive PRs... there are still strings hanging out here and there...